kafka_utils/kafka_check/commands/command.py

```diff
@@ -44,20 +44,26 @@ def run_command(self):
 
     def run(self, cluster_config, args):
         self.cluster_config = cluster_config
+        if args.broker_id == -1:
```

Comment:
I think we need to put this only for case with first_broker or controller. Because we also want to run this command without coordination, then we don't need broker_id and data_path